doc/html/classes.html

```diff
@@ -27,7 +26,7 @@
  <tr style="height: 56px;">
   <td id="projectalign" style="padding-left: 0.5em;" >
    <div id="projectname">NATS C Client with JetStream and Streaming support
-   &#160;<span id="projectnumber">3.8.0</span>
+   &#160;<span id="projectnumber">3.9.0-beta</span>
```

Comment:
This is a problem. The release process assumes that the version in the docs is `v3.9.0` and does not require regenerating the docs.

- (a) An easy solution would be to enable the `update-docs` job for the release branches, but that seems cumbersome.
- (b) A more "elegant" solution would be to do away with the `-beta` suffix, and set the next minor/major version in `main` "as is", e.g. `v3.9.0`. The downside is that a build off `main` looks like a "normal" version, even if unreleased.
- (c) restore the manual doxygen step in the release process

@kozlovic what do you think?

Reply:
There are two aspects of this: update of the documentation can be done at any time and reflect whatever is in the main branch, but that should not be reflected in the gh-pages branch, the one that is visible by users when clicking on the 'doc' badge. That one should only be updated when doing a release.
So the fact that the documentation in main is updated because of the switch to `v3.9.0-beta` is not a problem in my view, as long - again, that this is a different action than updating the gh-pages.

Reply:
I willdo (a) for now (auto-run this on `release_*`) branches, with accurate versioning. If it gets annoying, either improve the script or move back to running it manually.